packages/babel-plugin-proposal-decorators/src/transformer-2021-12.ts

```diff
@@ -1000,9 +997,6 @@ export default function (
   return {
     name: "proposal-decorators",
     inherits: syntaxDecorators,
-    manipulateOptions({ generatorOpts }) {
-      generatorOpts.decoratorsBeforeExport = decoratorsBeforeExport;
```

Comment:
I moved this to the syntax plugin, so that syntax options validation/handling is centralized.